docs/content/en/docs/reference/artifacts.md

```diff
@@ -475,11 +487,11 @@ These steps use `image-builder` to create an Ubuntu-based Amazon Machine Image (
 
    * `--os`: `ubuntu`
    * `--hypervisor`: For AMI, use `ami`
-   * `--release-channel`: Supported EKS Distro releases include 1-21, 1-22, 1-23 and 1-24.
+   * `--release-channel`: Supported EKS Distro releases include 1-21, 1-22, 1-23, 1-24 and 1-25.
```

Comment:
Same question about 1-21.

Reply:
Leaving this until deprecation in next minor release.